test/integration/users.test.js

```diff
@@ -169,13 +169,38 @@ describe("Users", function () {
         });
     });
 
+    it("Should get all the users with both archived true and false", function (done) {
+      chai
+        .request(app)
+        .get("/users")
+        .query({
+          q: "includes:archived",
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Users returned successfully!");
+          expect(res.body.users).to.be.a("array");
+          const userData = res.body.users;
+          userData.forEach((user) => {
+            expect(user.roles.archived).to.be.oneOf([false, true]);
+          });
```

Comment:
this doesn't guarantee that we are getting archived true users.  even if all users are not archived this will still pass.

Reply:
means if all users are archived false then it should also return those users too.
It will return a mix of both archived true and false.
**Expected behaviour**: will be when we pass this query we should get a mix of both archive true and false.
let me know if I missed something

Reply:
but we should also ascertain that at least we are getting 1 archived user and 1 non archived users in the list

Reply:
Okay got it 👍🏻

Reply:
Added the test case for this case.